roles/neutron-data-network/handlers/main.yml

```diff
@@ -14,3 +14,12 @@
 - name: restart xorp
   service: name=xorp state=restarted sleep=10
   ignore_errors: True
+
+- name: restart neutron lbaas f5 agent
+  service:
+    name: "{{ neutron.services.f5_oslbaasv2_agent.name }}"
+    state: restarted_if_running
+  when: restart|default('True') and ((neutron.lbaas.enabled == "smart" and
```

Comment:
change and conditions as list
when:
  - restart|default('True')
  - ((neutron.lbaas.enabled == "smart" and groups['controller'][0] not in groups['compute']) or neutron.lbaas.enabled|bool)
  - neutron.lbaas.f5.enabled|bool